### ISSUE TITLE
image and crop size in data_loader

### ISSUE THREAD

**Issue by zhihongp** [royorel/FFHQ-Aging-Dataset#14]
A bit confused by the data loading process.  It seems the downloaded data is 256x256, but applied with bilinear resizing to 513x513 in data_loader.py.  Is the resizing needed as the deeplab model is trained on 513x513 (or 512?) data?

**Comment by royorel** [comment]
@zhihongp You are correct, the deeplab network was trained on 513x513 images. (It was supposed to be 512x512, but we had a typo :) )

The data itself can be generated at any desired resolution, (see the readme file) but the segmentation network works on 513x513. The output segmentation maps are then resized with nearest neighbor interpolation.
